components/permit-holders/MedicalHistoryCard.tsx

```diff
@@ -35,33 +17,38 @@ const COLUMNS = [
     accessor: 'dateUploaded',
     disableSortBy: true,
     maxWidth: 200,
+    Cell: ({ value }) => {
+      return <Text>{new Date(value).toLocaleDateString('en-ZA')}</Text>;
+    },
   },
   {
-    accessor: 'associatedApplicationId',
+    accessor: 'associatedApplication',
     disableSortBy: true,
     minWidth: 180,
-    Cell: _renderConditionDetailsLink,
+    Cell: ({ value }) => {
+      return (
+        <MedicalHistoryModal application={value}>
+          <Link>
```

Comment:
We shouldn't use a link here as it does not take the user to a new page. Instead, this should be a button.